tls/s2n_x509_validator.c

```diff
@@ -648,6 +648,13 @@ static S2N_RESULT s2n_x509_validator_disable_time_validation(struct s2n_connecti
     return S2N_RESULT_OK;
 }
 
+#ifdef S2N_LIBCRYPTO_SUPPORTS_CUSTOM_OID
```

Comment:
I think this can be defined whether or not the `S2N_LIBCRYPTO_SUPPORTS_CUSTOM_OID` is enabled?